src/Config/upiconfig.xml

```diff
@@ -8,17 +8,17 @@
         <upi_attribute name='id' value='DYN'/>
         <upi_element name='level'>
           <upi_attribute name='name' value='release'/>
-          <upi_attribute name='id' value='2.17.0'/>
+          <upi_attribute name='id' value='2.17.1'/>
```

Comment:
release needs to remain `2.17.0`

Only the build id tag is supposed to be changed when incrementing the build part of the version